visualize-admin/visualization-tool
# fix: Concepts filter panel

Fixes #1704

This PR:
- adds a possibility to filter by three things at the same time (themes, organizations and termsets) – previously we could only have up to two filters,
- improves ordering logic of active filter panels,
- removes `AllTermsets` query in favor of fetching the termsets directly in `SearchCubes` query (and makes termsets filterable by themes and organizations).

## How to reproduce
1. Go to [this link](https://test.visualize.admin.ch/en/browse?dataSource=Prod).
2. Filter by `Cantons NFI` concept.
3. ❌ See that no cubes were returned, even though the filter said there would be 33 of them.
4. ❌ See that all other filters disappeared (themes, organizations).
5. ❌ See that `Concept` filter was not constrained to only show the selected concept.
6. Remove the filter.
7. Filter by any theme, then by any organization, and then by any concept.
8. ❌ See that the last concept selection removed all other filters.

## How to test
1. Go to [this link](https://visualization-tool-git-fix-filtered-termsets-count-ixt1.vercel.app/en/browse?dataSource=Prod).
2. Filter by `Cantons NFI` concept.
3. ✅ See that 33 cubes were returned, and that other filters were not removed, by rather filtered by the concept and show correct values (33).
4. ✅ See that `Concept` filter only shows selected value, matching the behavior of other filters.
5. ✅ See that you can narrow down the results by using other filters (including drafts and text search), and that they don't jump on top of the `Concept` filter, but keep their original positions.
